### PR TITLE
Online profile link

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Adds a profile link for Online programs. When the link field is set, a call to action will appear that links to the online program.

**Motivation and Context**
We want to be able to link to online programs from the main site degree profiles.

**How Has This Been Tested?**
Changes are available for review in dev.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: single-degree.php [hunk 1]
```diff
@@ -60,6 +60,25 @@
 				</div>
 			<?php endif; ?>
 			</div>
+			<?php echo main_site_degree_display_subplans( $post->ID ); ?>
+			<?php if ( isset( $post_meta['degree_online_url'] ) && ! empty( $post_meta['degree_online_url'] ) ) : ?>
+			<aside class="online-callout">
+				<a class="media-background-container d-block h-100 py-3 px-3 px-lg-4 hover-parent bg-inverse hover-text-black text-decoration-none" href="<?php echo $post_meta['degree_online_url']; ?>" target="_blank">
+					<img src="<?php echo $online_bg_img; ?>" alt="" class=" media-background object-fit-cover hover-child hover-child-scale-up" data-object-fit="cover">
```
Comment (cadie) [inline single-degree.php]: Looking at it in dev again after a while, what do you think about removing the `hover-child-scale-up` class? I'm thinking that just the color change might be enough for that callout.

File: single-degree.php [hunk 2]
```diff
@@ -60,6 +60,25 @@
...
+					<div class="media-background object-fit-cover bg-inverse-t-3 hover-child hover-child-hide fade" data-object-fit="cover"></div>
+					<div class="media-background object-fit-cover bg-primary-t-1 hover-child hover-child-show fade" data-object-fit="cover"></div>
+					<div class="row">
+						<div class="col-auto">
+							<span class="fa fa-info-circle fa-3x align-middle"></span>
+						</div>
+						<div class="col">
+							<p class="font-weight-black align-middle">Did you know this program is available online?</p>
```
Comment (cadie) [inline single-degree.php]: styling adjustment ideas: add `h5` class here in order to increase the font-size for that a smidge? We could remove the font-weight-black class with that addition

File: single-degree.php [hunk 3]
```diff
@@ -60,6 +60,25 @@
 				</div>
 			<?php endif; ?>
 			</div>
+			<?php echo main_site_degree_display_subplans( $post->ID ); ?>
+			<?php if ( isset( $post_meta['degree_online_url'] ) && ! empty( $post_meta['degree_online_url'] ) ) : ?>
+			<aside class="online-callout">
+				<a class="media-background-container d-block h-100 py-3 px-3 px-lg-4 hover-parent bg-inverse hover-text-black text-decoration-none" href="<?php echo $post_meta['degree_online_url']; ?>" target="_blank">
```
Comment (cadie) [inline single-degree.php]: I'm going back and forth on updating this if the 'Did you know' text is updated to an `h5` class, but we could increase the vertical padding for large+ displays here to `py-lg-4` to give it a little extra room without it looking too big

File: single-degree.php [hunk 4]
```diff
@@ -60,6 +60,25 @@
 				</div>
 			<?php endif; ?>
 			</div>
+			<?php echo main_site_degree_display_subplans( $post->ID ); ?>
+			<?php if ( isset( $post_meta['degree_online_url'] ) && ! empty( $post_meta['degree_online_url'] ) ) : ?>
+			<aside class="online-callout">
```
Comment (cjg89) [inline single-degree.php]: Need to add a unique `aria-label` or `aria-labelledby` to this `aside`.

File: single-degree.php [hunk 5]
```diff
@@ -60,6 +60,25 @@
...
+				<a class="media-background-container d-block py-3 px-3 px-lg-4 py-lg-4 hover-parent bg-inverse hover-text-black text-decoration-none" href="<?php echo $post_meta['degree_online_url']; ?>" target="_blank">
+					<img src="<?php echo $online_bg_img; ?>" alt="" class=" media-background object-fit-cover hover-child" data-object-fit="cover">
+					<div class="media-background object-fit-cover bg-inverse-t-3 hover-child hover-child-hide fade" data-object-fit="cover"></div>
+					<div class="media-background object-fit-cover bg-primary-t-1 hover-child hover-child-show fade" data-object-fit="cover"></div>
+					<div class="row">
+						<div class="col-auto">
+							<span class="fa fa-info-circle fa-3x align-middle"></span>
```
Comment (cjg89) [inline single-degree.php]: Span needs `aria-hidden="true"`

**Pull request merged**